_docs/01-getting-started.md

```diff
@@ -12,6 +12,7 @@ Clone the Github repo, and then open the Xcode project.
 $ git clone https://github.com/facebook/componentkit
 $ cd componentkit/Examples/WildeGuess/
 $ open WildeGuess.xcodeproj
+$ carthage install
```

Comment:
Thanks for adding this!
Let's move it to the first step and alter comment at line 9 to reflect this as well